### DISCUSSION TITLE
Distance between staves

### DISCUSSION THREAD

**Discussion by gerardoscoppetta** [opensheetmusicdisplay/opensheetmusicdisplay#1476] [Q&A]
Is there a way to manually set the vertical distance between the staves? Thank you
![distance_staff](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/41266915/77aa1241-d0c7-4fe5-a7e8-64617a0e6547)

**Comment by sschmidTU** [comment]
Handy that I just answered the exact opposite of this question a few days ago ;)
https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/issues/1475

You probably want to increase the distance variables, e.g.:
```js
osmd.EngravingRules.MinSkyBottomDistBetweenSystems = 12
```

(I think the other EngravingRules don't necessarily take the skyline/bottomline into account, meaning how much the notes extend beyond the 5 note lines / stafflines)

**Reply by gerardoscoppetta** [reply]
Tried it immediately and it works very well.
A big thanks :)
